enterprise/admin/__init__.py

```diff
@@ -101,6 +101,22 @@ def identity_provider(instance):
         ec_idp = instance.enterprise_customer_identity_provider
         return ec_idp and ec_idp.provider_name or ec_idp.provider_id
 
+    def enterprise_catalog(self, instance):
+        """
+        Instance is EnterpriseCustomer.
+
+        Return catalog id with catalog details link to display in enterprise customer list view.
+        """
+        catalog_url = get_catalog_admin_url_template().format(catalog_id=instance.catalog)
```

Comment:
See note in enterprise.utils below -- I think it would be better to pass the catalog_id to this util method instead of doing the format here.